### PR TITLE
Adds log out functionality to the layout

### PR DESCRIPTION
**Pull request by ghatighorias**

### CODE REVIEW

**Comment by ghatighorias** [comment]
I am not sure exactly what is the best way of showing the user and is log-out link. tell if there is any idea

File: web/controllers/coherence/session_controller.ex [hunk 1]
```diff
@@ -0,0 +1,323 @@
+defmodule CoursePlanner.Coherence.SessionController do
```
Comment (ggpasqualino) [inline web/controllers/coherence/session_controller.ex]: we shouldn't add this file

I checked it out and the reason you need to add it is that we have the coherence protected routes wrong.

Could you move
```elixir
coherence_routes :protected
```
into it's own block?
```elixir
  scope "/" do
    pipe_through :protected
    coherence_routes :protected
  end
```
then remove this file?

**Comment by ggpasqualino** [comment]
> I am not sure exactly what is the best way of showing the user and is log-out link. tell if there is any idea

@ghatighorias that's fine for now, later there will be lateral navigation bar and a header with user info and logout link

File: web/controllers/coherence/session_controller.ex (hunk 1 above)
Comment (ghatighorias) [inline web/controllers/coherence/session_controller.ex]: cool, 
adding the session_controller was the only option i have found but it was pretty ugly

**Pull request merged**